backend/ml/management/commands/ml_train.py

```diff
@@ -133,11 +150,20 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
         criterias_list = Poll.default_poll().criterias_list
-        comparison_data = fetch_data()
+        comparison_data_trusted = fetch_data()
+        compraison_data_not_trusted = fetch_data(trusted=False)
         if TOURNESOL_DEV:
             logging.error('You must turn TOURNESOL_DEV to 0 to use this')
         else:  # production mode
-            glob_scores, loc_scores = ml_run(
-                comparison_data, criterias=criterias_list, save=True, verb=-1
+            # Run for trusted users
+            glob_scores_trusted, loc_scores_trusted = ml_run(
+                comparison_data_trusted, criterias=criterias_list, save=True, verb=-1
             )
-            save_data(glob_scores, loc_scores)
+            save_data(glob_scores_trusted, loc_scores_trusted)
+
+            # Run for all users including non trusted users
+            glob_scores_not_trusted, loc_scores_not_trusted = ml_run(
+                compraison_data_not_trusted, criterias=criterias_list, save=True, verb=-1
+            )
+            save_data(glob_scores_not_trusted, loc_scores_not_trusted, trusted=False)
```

Comment:
Currently you have implemented the logic with

Pseudo code:
```
def handle():
    fetch_trusted
    fetch_all
    run_ml_trusted
    save_trusted
    run_ml_all
    save_all
```

What do you think of the refactor:

```
def process():
    fetch
    run_ml
    save

def handle():
    process_all
    process_trusted
```